ui/src/app/Deployment/DeploymentProgress/CancelDeployment.tsx

```diff
@@ -14,7 +14,11 @@ export const CancelDeployment = () => {
     try {
       const cancelled = await cancelDeployment()
       // TODO add visual confirmation that deployment was cancelled
-      console.log(`Deployment was cancelled: ${cancelled}`)
+      console.log(`Deployment was cancelled: ${cancelled}`);
+      setCancelled(true);
+      document.getElementsByClassName("cancelButton")[0].remove();
```

Comment:
What are these for?

Reply:
so when we cancel a deployment, this hides the buttons and the user would only see that the deployment has been cancelled.

Reply:
While this works, it will be cleaner to have React remove the element (hide it) instead of using low level API to remove an element. 
It's fine leaving this one in here until we have the changes for proper cancelling of deployment and then we can implement hiding the buttons when deployment is marked cancelled.